Add aggregate to migration guide deprecations (#246)
neo4j/docs-graphql: modules/ROOT/pages/migration/index.adoc

=== Deprecated aggregations on ID fields

Aggregations on ID fields are now deprecated and will be removed in the future.

> === Deprecated Aggregate operations
> 
> Explicit Aggregate operations, both top level and nested, have been deprecated in favor of the fields `aggregate` inside Connection operations.

Reviewer: ```suggestion
=== Deprecated aggregation operations

Explicit aggregation operations, both top-level and nested, have been deprecated in favor of the `aggregate` field inside connection operations.
```

> So, for example:

Reviewer: ```suggestion
For example:
```

[source, graphql, indent=0]
----
query {
  moviesAggregate {
    count
    title {
      shortest
    }
  }
}
----

> Is deprecated in favor of:

Reviewer: ```suggestion
The previous query is deprecated in favor of the following:
```

[source, graphql, indent=0]
----
query {
  moviesConnection {
    aggregate {
      node {
        title {
          shortest
        }
      }
      count {
        nodes
      }
    }
  }
}
----

> Note that `count` now returns the explcit fields `nodes` and `edges`.

Reviewer: ```suggestion
Note that `count` now returns the explicit fields `nodes` and `edges`.
```
